torch/csrc/jit/frontend/tracer.cpp

```diff
@@ -964,11 +966,144 @@ void setWarn(warn_fn_type fn) {
   warn_callback.store(fn);
 }
 } // namespace tracer
+
+void tracingFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
```

Comment:
I noticed this function lived in an anonymous namespace before (in register_c10_ops.cpp), is that necessary?